src/index.js

```diff
@@ -31,4 +27,6 @@ app.post('/api/chat', async (c) => {
 	});
 });
 
+app.get('/*', (c) => c.env.ASSETS.fetch(c.req.raw));
```

Comment:
For reference: https://developers.cloudflare.com/pages/functions/advanced-mode/#set-up-a-function